R/mergedatasetsbycase.R

```diff
@@ -1936,9 +1936,11 @@ mergedDataSet <- function(data.sets, matched.names, merged.names,
     merged.data.set <- data.frame(merged.data.set, check.names = FALSE)
     names(merged.data.set) <- merged.names
 
-    mergesrc.name <- uniqueName("mergesrc", names(merged.data.set), "_")
-    merged.data.set[[mergesrc.name]] <- mergeSrc(n.data.set.cases,
-                                                 names(data.sets))
+    mergesrc.exists <- "mergesrc" %in% merged.names
+    existing.mergesrc <- if (mergesrc.exists) merged.data.set[["mergesrc"]] else NULL
```

Comment:
I think these two lines can be removed and `existing.mergesrc` below can be replaced with `merged.data.set[["mergesrc"]]` since it would be `NULL`  if `"mergesrc"` isn't present